### PR TITLE
feat: support session variables in sql client

### LINKED ISSUE #714: Support set execute_mode in CLI
We may support setting `execute_mode` as `online` or `offline` in command-cli and SQL clients.

### PR DESCRIPTION
**Pull request by vagetablechicken**
Closes #714 
Current CLI session variables: execute_mode, performance_sensitive.
P.S.  'performance_sensitive' will set the value to sql cluster router. 'execute_mode' only supports set/show, no effect.

### CODE REVIEW

File: src/cmd/sql_cmd.h [hunk 1]
```diff
@@ -74,6 +74,9 @@ const std::string VERSION = std::to_string(OPENMLDB_VERSION_MAJOR) + "." +  // N
 std::string db = "";  // NOLINT
 ::openmldb::sdk::DBSDK* cs = nullptr;
 ::openmldb::sdk::SQLClusterRouter* sr = nullptr;
+using VariableMap = std::map<std::string, std::string>;
+VariableMap session_variables = {VariableMap::value_type("execute_mode", "online"),
+                                 VariableMap::value_type("performance_sensitive", "true")};
```
Comment (jingchen2222) [inline src/cmd/sql_cmd.h]: Actually, we will eliminate performance-sensitive mode in #877 .

Comment (jingchen2222) [reply src/cmd/sql_cmd.h]: We have to resolve conflicts after #796 has been merged.

Comment (vagetablechicken) [reply src/cmd/sql_cmd.h]: OK, I'll fix it.

File: src/cmd/sql_cmd.h [hunk 2]
```diff
@@ -607,6 +610,16 @@ void HandleCmd(const hybridse::node::CmdPlanNode* cmd_node) {
             }
             break;
         }
+        case hybridse::node::kCmdShowSessionVariables: {
+            for (auto& pair : session_variables) {
+                std::cout << pair.first << "=" << pair.second << std::endl;
```
Comment (tobegit3hub) [inline src/cmd/sql_cmd.h]: Print the variables like MySQL does, or simply with headers.

```
mysql> show variables like "window%";
+------------------------------+-------+
| Variable_name                | Value |
+------------------------------+-------+
| windowing_use_high_precision | ON    |
+------------------------------+-------+
1 row in set (0.00 sec)
```

**Review by tobegit3hub — COMMENTED** [review]
Others LTGM

File: src/cmd/sql_cmd.h (hunk 2 above)
Comment (vagetablechicken) [reply src/cmd/sql_cmd.h]: If we move session variables into sql router, we will get `ResultSet`, and we can print `ResultSet` like this. I think it's no need to make the map print look like ResultSet.

Comment (vagetablechicken) [reply src/cmd/sql_cmd.h]: Or we can convert the session variable map to ResultSet? We can reuse it later.

Comment (vagetablechicken) [reply src/cmd/sql_cmd.h]: We use `PrintItemTable` to print it temporarily.

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by aceforeverd — DISMISSED** [review]
Resolve failed tests

**Review by dl239 — APPROVED** [review]

**Comment by tobegit3hub** [comment]
The case `ast_node_converter_test` will always fail in MacOS.

**Comment by vagetablechicken** [comment]
[data import for offline and online in the task manager](https://trello.com/c/OqReyYcD/23-data-import-for-offline-and-online-in-the-task-manager)

**Pull request merged**